### PR TITLE
Better mechanism to detect the width of the dom element to prevent it…

### PR DESCRIPTION
**Pull request by adieyal**
## Description

Previously the downloaded image would cut off the bottom text in the image. Now using the client bounding box to detect the actual height of the image

## Related Issue
N/A

## How to test it locally
Use the gcro backend. Download the age of respondents chart. The bottom text should be full displayed

### Updated
Using `getBoundingClientRect` instead of `.width()` to detect the height and height of the element being downloaded 

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [ ]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [ ]  ⚙️ ran jslint
- [ ]  🧰 ran codeclimate locally

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/reusable-charts/horizontal-bar-chart.js [hunk 1]
```diff
@@ -211,19 +211,22 @@ export function horizontalBarChart() {
     }
 
     chart.saveAsPng = function (container) {
+
         let element = $(container).closest('.profile-indicator')[0];
+
         $(element).find('.profile-indicator__options').attr('data-html2canvas-ignore', true);
         $(element).find('.profile-indicator__filters').attr('data-html2canvas-ignore', true);
+        const rightMargin = 60;
```
Comment (milafrerichs) [inline src/js/reusable-charts/horizontal-bar-chart.js:219]: nice refactor removing a magic number.

**Review by milafrerichs — APPROVED** [review]
